internal/network/command_private_link_access.go

```diff
@@ -0,0 +1,130 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type privateLinkAccessCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
```

Comment:
Any reason we aren't using the struct from the parent command?

Reply:
Ah, for future reference @MuweiHe we prefer defining as few structs as possible. If the parent command uses the same type of authentication as its children, the same struct can be used for all of them.

Reply:
Thank you! Updated the private link access command to use the struct from the parent command.

I will update the rest in a separate PR after merging this PR and https://github.com/confluentinc/cli/pull/2249